src/API/REST/V3/Routes/Donors/DonorStatisticsController.php

```diff
@@ -91,7 +91,13 @@ public function get_item($request)
                 'lifetimeAmount' => $query->getLifetimeDonationsAmount(),
                 'highestAmount' => $query->getHighestDonationAmount(),
                 'averageAmount' => $query->getAverageDonationAmount(),
+                'firstDonation' => $query->getFirstDonation(),
+                'donationCount' => $query->getDonationsCount(),
+                'lastContribution' => $query->getLastContribution()
             ],
+            'donorSince' => $donor->createdAt->format('Y-m-d H:i:s'),
```

Comment:
Is this a statistic? I would think we could just get the date the donor was createdAt from the donor endpoint/entity